ledger-core/conveyor/smachine/slot_aliases.go

```diff
@@ -55,13 +55,14 @@ func (a *slotAliases) RetainAll(parent *slotAliases, removeFn func(k interface{}
 			}
 		}
 	}
+
+	keys := a.keys[i:]
```

Comment:
????

Reply:
!!!!

Reply:
There was a bug in a part where cleanup was processed, so it's @cyraxred fix for that problem

Reply:
it is ;)